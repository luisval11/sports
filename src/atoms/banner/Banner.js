import React from "react";
import { Card } from "react-bootstrap";

export const Banner = () => {
	return (
		<Card className="m-3 text-center">
			<Card.Body>
				<Card.Title>Football World Cup Score Analysis Board</Card.Title>
			</Card.Body>
		</Card>
	);
};
