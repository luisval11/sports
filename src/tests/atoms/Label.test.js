import { mount, shallow } from "enzyme";
import { Banner } from "../../atoms/banner/Banner";
import LabelI from "../../atoms/label/Label";

describe("Test on Label", () => {
	test("should match initial snapshot", () => {
		let wrapper = shallow(<LabelI text="Label"/>);
		expect(wrapper).toMatchSnapshot();
	});

    test("should has the text 'Label'", () => {
        const text = "Label";
		let wrapper = mount(<LabelI text={text}/>);
        expect(wrapper.find('.form-label').text().trim()).toBe(text);
	});
    
});
