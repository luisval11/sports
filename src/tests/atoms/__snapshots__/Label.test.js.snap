// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on Label should match initial snapshot 1`] = `
<FormLabel
  column={false}
  visuallyHidden={false}
>
  Label
</FormLabel>
`;
